lua/telescope/builtin/files.lua

```diff
@@ -368,6 +368,11 @@ files.current_buffer_fuzzy_find = function(opts)
     })
   end
 
+  local ts_ok, ts_parsers = pcall(require, 'nvim-treesitter.parsers')
+  if ts_ok then
+    filetype = ts_parsers.ft_to_lang(filetype)
+  end
+
   local ok, parser = pcall(vim.treesitter.get_parser, bufnr, filetype)
   if ok then
     local query = vim.treesitter.get_query(filetype, "highlights")
```

Comment:
I suppose the other thing is that we should pcall here as well -- I didn't realize this would fail if it didn't find anything.

If we do `local query_ok, query = pcall(...)` here, then that would be double safe.

If you add that in I'll merge.

Thanks for the PR :)

Reply:
Yeah, that sounds good. Got it done :)